src/admin/components/BannerTable/ActionButtons.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  Icon,
+  Divider,
+  Responsive,
+  Popup,
+  Button,
+  Table,
+} from 'semantic-ui-react';
+
+const ActionButtons = ({handleDelete, handleEdit}) => {
```

Comment:
Since you don't have any state for this component, you can return the JSX as a functional component:

```js
const ActionButtons = ({handleDelete, handleEdit}) => (
    <Table.Cell>
        ...
    </Table.Cell>
)
```